tuplex/core/src/physical/TransformStage.cc

```diff
@@ -848,7 +850,7 @@ namespace tuplex {
         }
 
         // check symbols are valid...
-        if(!(_syms->functor && _syms->initStageFunctor && _syms->releaseStageFunctor)) {
+        if(!((_syms->functor || _syms->functorWithExp) && _syms->initStageFunctor && _syms->releaseStageFunctor)) {
```

Comment:
this check seems somehow weird. I think you want to check that in the case of updateInputExceptions==true, also functorWithExp is true (plus the other case)